RentACar/RentACar/Repositories/ClientCRUD.cs

```diff
@@ -0,0 +1,54 @@
+﻿using RentACar.Models;
+using RentACar.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RentACar.Repositories
+{
+    public class ClientCRUD : Repository<Client>, IClientCRUD
```

Comment:
I like this change from CRUD to a Repository pattern!!! But let's go full speed on it: rename the interfaces and drop the CRUD wording from all the names, eg.: this is no longer a ClientCRUD but a ClientRepository <3

Reply:
oh yeah! now this sounds like there is not so much consistency of names! i will change it, thanks!